modules/snooker/src/main/ts/ephox/snooker/api/Generators.ts

```diff
@@ -166,12 +171,55 @@ const merging = function (generators: Generators) {
       // Remove any width calculations because they are no longer relevant.
       Css.remove(raw, 'width');
       Css.remove(cell, 'width');
+
+      Attribute.getOpt(cell, 'scope').each((scope) => Attribute.set(raw, 'scope', scope));
+
       return raw;
     };
   };
 
+  const merge = (cells: SugarElement[]) => {
+    const getScopeProperty = () => {
+      const attributeOpts = Arr.map(cells, (cell) => {
+        const attributeOpt = Attribute.getOpt(cell, 'scope');
+
+        return attributeOpt.map((attribute) => attribute.substr(0, 3));
```

Comment:
I'm assuming this turns `colgroup` into `col` and `rowgroup` into `row` but can we please comment it